commands.go

```diff
@@ -114,6 +117,31 @@ func (b *bot) sudoku(m dggchat.Message, s *dggchat.Session) {
 	s.SendMute(m.Sender.Nick, -1)
 }
 
+func (b *bot) frenchToastAlert(m dggchat.Message, s *dggchat.Session) {
+	type ftlXML struct {
+		XMLName xml.Name `xml:"frenchtoast"`
+		Status  string   `xml:"status"`
+	}
+	if !strings.HasPrefix(m.Message, "!frenchToastAlert") {
```

Comment:
If we were to accept this (still undecided) I think I would prefer `!frenchtoast`